src/lib/application/services/GitTriageService.ts

```
@@ -0,0 +1,404 @@
+      until: until.toISOString(),
+      maxCount: options?.maxCommits,
+      cwd,
+    });
+
+    if (rawCommits.length === 0) {
+      return this.emptyResult(startTime, tagInfo);
+    }
+
+    // Build a map of SHA -> position for tag adjacency
+    const shaToPosition = new Map<string, number>();
+    rawCommits.forEach((c, i) => shaToPosition.set(c.sha, i));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)